pycocoevalcap/spice/spice.py

```diff
@@ -85,7 +85,7 @@ def compute_score(self, gts, res):
         for image_id in imgIds:
           # Convert none to NaN before saving scores over subcategories
           score_set = {}
-          for category,score_tuple in imgId_to_scores[image_id].iteritems():
+          for category,score_tuple in imgId_to_scores[image_id].items():
             score_set[category] = {k: self.float_convert(v) for k, v in score_tuple.items()}
```

Comment:
score_set[category] = {k: self.float_convert(v) for k, v in list(score_tuple.items())}